pkg/genericapiserver/genericapiserver.go

```diff
@@ -583,10 +583,24 @@ func verifyEtcdServersList(options *options.ServerRunOptions) {
 	}
 }
 
+func verifySecurePort(options *options.ServerRunOptions) {
+	if options.SecurePort < 1 || options.SecurePort > 65535 {
+		glog.Fatalf("--secure-port %v must be between 1 and 65535, inclusive", options.SecurePort)
+	}
+}
+
+func verifyInsecurePort(options *options.ServerRunOptions) {
+	if options.InsecurePort < 1 || options.InsecurePort > 65535 {
```

Comment:
Reasonable. I will add it. Thanks @lavalamp !
I just find that the flag `--kubernetes-service-node-port` also has to be verified, the condition in current `verifyServiceNodePort` is not enough. I will fix it immediately and commit to this PR.